src/main/java/seedu/address/ui/GoodInformation.java

```diff
@@ -30,6 +31,20 @@ public GoodInformation(Good good, int displayedIndex) {
         id.setText(displayedIndex + ". ");
         goodName.setText(good.getGoodName().toString());
         goodQuantity.setText(good.getGoodQuantity().toString());
+        warningLowQuantity(good.getGoodQuantity(), good.getThreshold());
+    }
+
+    /**
+     * Sets an alert background for good with quantity lower or equals to threshold.
+     * @param quantity  quantity of good in inventory.
+     * @param threshold threshold quantity of good.
+     */
+    private void warningLowQuantity(GoodQuantity quantity, GoodQuantity threshold) {
+        if (quantity.goodQuantity <= threshold.goodQuantity) {
```

Comment:
Want to define a method to compare in the class?

Reply:
i forget that i have already defined the method in the class